astropy/nddata/tests/test_bitmask.py

```diff
@@ -419,10 +419,10 @@ def test_map_add_flags():
     assert map2.DEAD.__doc__ == "a really dead pixel"
     assert map2.DEAD == 4
 
-    map2 = map1 + [("HOT", 2), ("DEAD", 4)]
+    map2 = [*map1, ("HOT", 2), ("DEAD", 4)]
```

Comment:
I don't think this is a concatenation either.

Reply:
No. It's something much weirder.